src/components/BlogComponents/BlogHero.jsx

```diff
@@ -3,27 +3,45 @@ import styled from 'styled-components';
 
 import Hero from '../../components/Hero';
 import Container from '../../components/Container';
-import { H1 } from '../../components/Typography';
+import { H2 } from '../../components/Typography';
 import { theme } from '../../utils/styles';
 
 const StyledHero = styled(Hero)`
   ${Container} {
-    padding-bottom: 2.5em;
+    padding-bottom: 4em;
   }
+  ${({ theme }) => theme.media.large`
+    ${Container} {
+      padding-bottom: 6em;
+    }
+  `}
   ${({ theme }) => theme.media.xlLarge`
-    background-position: center -670px;
-
+    background-position: center -425px;
```

Comment:
the smaller heading and this position makes the top look a lot less interesting with this change.

Reply:
Looks like a lot of these headings are less interesting now :)

Reply:
The heading/subheading font size/weight/line-height is pretty much the same as the current site, give or take some adjustments in the spacing between and around the new h2/p tags. Are you seeing a heading that's become visibly smaller somewhere?

The adjusted positioning was done for visibility/legibility (because putting imagery behind text generally makes it harder to read and we've already chosen colors for the hero sections that don't really provide much contrast anyway) and just for general consistency across pages.

Reply:
Below is how it looks like for me, compared to current. Though, I agree with the accessibility thing. Perhaps, there's a place where we make it more accessible but also as visually interesting as before.

Current:
![current blog header](https://user-images.githubusercontent.com/535884/162824306-4536bb61-1d12-4f23-9ca1-6d742bdd183e.png)

New:
![new blog header](https://user-images.githubusercontent.com/535884/162824349-07cfbcb1-bd5b-467e-8eed-bdc84fe69143.png)

Reply:
I can nudge it up to get that starfish and crab back in play but they're staying well away from that heading, haha.

Reply:
I think that's reasonable!